### DISCUSSION TITLE
Scrollback buffer is limited to `u32::MAX` bytes

### DISCUSSION THREAD

**Discussion by lopopolo** [ghostty-org/ghostty#3884] [Feature Requests, Ideas] [answered]
Results in a config parsing error:

```
scrollback-limit = 4294967296
```

Success:

```
scrollback-limit = 4294967295
```

Since this parameter directly influences allocating memory, I'd expect it to be bounded by `size_t`, or maybe something like Rust's `isize::MAX` which echos the LLVM max allocation size.

**Comment by mitchellh** [comment, marked as answer]
Approved #3900
